ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManagerMetrics.java

```diff
@@ -190,10 +190,8 @@ void initializePartitionMetrics() {
           }
         }
         if (downReplicaCount > replicaCount / 2) {
-          if (logger.isTraceEnabled()) {
-            logger.trace("There are more than more of the replicas are down for partition {}, the down replicas are {}",
+            logger.info("There are more than more of the replicas are down for partition {}, the down replicas are {}",
```

Comment:
This msg doesn't make sense - there are more than more of replicas are down ??

Reply:
updated